build.gradle

```diff
@@ -76,6 +76,7 @@ jar {
 }
 
 configurations {
+    all*.exclude module : 'spring-boot-starter-logging'
```

Comment:
The dependencies I've addes (postgres and spring-boot-starter-jdbc) have nothing to do with logging. But the application stopped launching with logging conflicts after I've added then. I'm not sure if is it ok to remove this logging.